code/modules/jobs/job_types/security.dm

```diff
@@ -124,7 +124,7 @@ Detective
 
 	outfit = /datum/outfit/job/detective
 
-	access = list(access_sec_doors, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_brig, access_weapons)
+	access = list(access_sec_doors, access_security, access_forensics_lockers, access_morgue, access_maint_tunnels, access_court, access_brig, access_weapons)
```

Comment:
i had added the detective skeleton crew access to security for the fact that security players make up a tiny fraction of cit playerbase, let alone regular ones, so when we get secret with only a detective in sec it makes it easier on the crew so they don't get steam rolled by some guy hacking into the armoury five seconds in and killing them all